nilearn/datasets/struct.py

```diff
@@ -535,45 +545,45 @@ def _fetch_surf_fsaverage5(data_dir=None, url=None, resume=True, verbose=1):
     fdescr = _get_dataset_descr(dataset_name)
 
     # Download fsaverage surfaces and sulcal information
-    surf_file = '%s.%s.gii'
-    surf_url = url + '%i/%s.%s.gii'
-    surf_nids = {'lh pial': 9344, 'rh pial': 9345,
-                 'lh infl': 9346, 'rh infl': 9347,
-                 'lh sulc': 9348, 'rh sulc': 9349}
+    pial_file = '%s.%s.gii.gz'
+    pial_path = os.path.join(FSAVERAGE5_PATH, pial_file)
+
+    sulc_file = '%s.%s.gii'
+    sulcal_url = url + '%i/%s.%s.gii'
+    sulcal_nids = {'lh sulc': 9348, 'rh sulc': 9349}
 
     pials = []
     infls = []
     sulcs = []
     for hemi in [('lh', 'left'), ('rh', 'right')]:
-
-        pial = _fetch_files(data_dir,
-                            [(surf_file % ('pial', hemi[1]),
-                                surf_url % (surf_nids['%s pial' % hemi[0]],
-                                            'pial', hemi[1]),
-                              {})],
-                            resume=resume, verbose=verbose)
-        pials.append(pial)
-
-        infl = _fetch_files(data_dir,
-                            [(surf_file % ('pial_inflated', hemi[1]),
-                              surf_url % (surf_nids['%s infl' % hemi[0]],
-                                          'pial_inflated', hemi[1]),
-                              {})],
-                            resume=resume, verbose=verbose)
-        infls.append(infl)
-
+        # pial
+        new_pial_path = os.path.join(data_dir,
+                                     pial_file % ('pial', hemi[1]))
+        shutil.copy(pial_path % ('pial', hemi[1]), new_pial_path)
+        _uncompress_file(new_pial_path, verbose=0)
```

Comment:
I would not copy or unzip these files, I would just return the path to data shipped with nilearn.

Reply:
But, then you if you return gzipped files as it is. How would you manage to plot with them ? AFAIK they needs to be unzipped again for nibabel to read them. Let me know if did not understand your comment correctly.

Reply:
you're right. This may be seen as a shortcoming of nilearn.surface.load_suf_mesh, maybe it should be able to uncompress them on the fly (load_img accepts .nii.gz, so maybe load_surf_mesh should accept .gii.gz)

Reply:
but as you said, as it stands now, it needs to be uncompressed before, as you did in the fetcher

Reply:
Return the files as it is. Uncompress them using load_surf_mesh ?

Reply:
I think it is a good idea. I can do in this PR. When you uncompress them using load_surf_mesh, it should be uncompressed on the fly ? Do you know how to do that ? Using gzip ?

Reply:
Ok, here is how you can do it:
```
>>> import gzip
>>> from nibabel import gifti
>>>
>>> with gzip.open('nilearn/datasets/data/fsaverage5/pial.left.gii.gz') as f:
...     as_bytes = f.read()
...     
>>> parser = gifti.GiftiImage.parser()
>>> parser.parse(as_bytes)
>>> coords, faces = (a.data for a in parser.img.darrays)
>>> print('coords: {}; faces:{}'.format(coords.shape, faces.shape))
coords: (10242, 3); faces:(20480, 3)
>>> 
```

Reply:
That's great! Thanks

Reply:
but if this seems scary a safer option may be to uncompress it to a temporary file, read it from there, and copy the result

Reply:
No this is great. Just that I need to check if this snippet supports all versions of nibabel.

Reply:
> Not touching the disk is always more robust. You never know in what kind of environment people might run us.

great, in this case please just see the updated comment above to be robust to different flavors of gifti files (or at least fail consistently with the uncompressed version :) )

Reply:
>once you have parser.img, do the same as what is done for gii files

Did you see getting a parser.img for nibabel 2.0.2 ?